localization/english/CWP/CWP_production_methods_l_english.yml

```diff
@@ -902,14 +910,63 @@
   pmg_ownership_telecommunications_industry: "Ownership"
   pmg_base_tourism: "Tourism"
   pmg_ownership_tourism: "Ownership"
-  pmg_base_financial_services: "Financial Services"
+  pmg_base_financial_services: "Personal Services"
```

Comment:
Why was this changed?

Reply:
To differentiate personal financial services and the new commercial services PMG.